lib/ViroDB/ResultSet/PrimerSearch.pm

```diff
@@ -0,0 +1,56 @@
+use strict;
+use warnings;
+use 5.018;
+use utf8;
+
+package ViroDB::ResultSet::PrimerSearch;
+use Moose;
+use MooseX::NonMoose;
+use MooseX::MarkAsMethods autoclean => 1;
+extends 'ViroDB::ResultSet';
+
+with 'ViroDB::Helper::ResultSet::SearchArrayOverlaps';
+with 'ViroDB::Helper::ResultSet::SearchFreeform', {
+    text_fields => [qw[
+        name
+        sequence
+        orientation
+        notes
+        date_added
```

Comment:
I don't think we want date in the freeform search, it's not going to be relevant to most users and could interfere with useful name or position results

Reply:
(I'm not sure how I feel about the inclusion of `sequence` either but it's likely more useful than date added..)